templates/paragraphs/txex_paragraph_42/paragraph--txex-paragraph-42.html.twig

```diff
@@ -0,0 +1,10 @@
+{% extends "paragraph.html.twig" %}
```

Comment:
I'm not sure where this template gets used, but just wrapping `{{ content }}` doesn't seem right to me.